--readme fix (#23)
meta-llama/llama3: README.md

Pre-requisites: Make sure you have `wget` and `md5sum` installed. Then run the script: `./download.sh`.

> for windows get it from here : [[gnu wget]](https://eternallybored.org/misc/wget/)
> 
> for mac just run :
> if you have homebrew : 
> ```
> brew install wget
> 
> ```
> 

Reviewer: ```suggestion
If you use Windows, you can get download [gnu wget](https://eternallybored.org/misc/wget/). For macOS, run `brew install wget`.
```

(I believe most Mac users are familiar with Homebrew these days)

- Visit one of the repos, for example [meta-llama/Meta-Llama-3-8B-Instruct](https://huggingface.co/meta-llama/Meta-Llama-3-8B-Instruct).
- Read and accept the license. Once your request is approved, you'll be granted access to all the Llama 3 models. Note that requests use to take up to one hour to get processed.

> - To download the original native weights to use with this repo, click on the "Files and versions" tab and download the contents of the `original` folder. You can also download them from the command line if you `pip install huggingface-hub`:

Reviewer: 🤦
